src/SFA.DAS.ApplyService.InternalApi.UnitTests/GatewayOrganisationChecksControllerTests.cs

```diff
@@ -117,5 +120,21 @@ public void get_website_address_manually_entered_returns_no_value_when_not_prese
 
             Assert.IsNull(actualResult);
         }
+
+
+        [Test]
```

Comment:
This appears to only cover one of the three possible scenarios when looking up the website address

Reply:
One more test scenario added

Reply:
The code is handling/logging a message when no website is found - that was the third scenario I was referring to

Reply:
All scenarios now covered.